orchagent/intfsorch.cpp

```diff
@@ -1319,6 +1319,15 @@ bool IntfsOrch::removeRouterIntfs(Port &port)
     const auto id = sai_serialize_object_id(port.m_rif_id);
     removeRifFromFlexCounter(id, port.m_alias);
 
+    for (auto it = m_rifsToAdd.begin(); it != m_rifsToAdd.end(); ++it)
```

Comment:
a minor comment. no need to call `removeRifFromFlexCounter` if the object is in the list

Reply:
done